tests/multihost/prepare/main.fmf

```diff
@@ -1,2 +0,0 @@
-summary: Verify that multihost preparation works
-duration: 10m
```

Comment:
Coverage from this test has been moved to `/tests/multihost/complete`?

Reply:
Correct, `/tests/multihost/complete` checks `prepare` step as well.

Reply:
Good, thanks for clarification.